vale/Grafana/Headings.yml

```diff
@@ -24,7 +25,10 @@ exceptions:
   - CPU
   - eBPF
   - Frontend Observability
+  - Generic
```

Comment:
I wonder if _Generic_ should be an exception in of itself. Is it part of a multi-word term that is used in headings?

Reply:
Yes, `Generic OAuth`, which is the name of the integaration.

Reply:
How about:

```suggestion
  - Generic OAuth
```